devU-api/src/entities/deadlineExtensions/deadlineExtensions.validator.ts

```diff
@@ -0,0 +1,16 @@
+import { check } from 'express-validator'
+
+const creatorId = check('creatorId').isNumeric()
+const userId = check('userId').isNumeric()
+const assignmentId = check('assignmentId').isNumeric()
+const deadlineDate = check('deadlineDate').trim().isISO8601().toDate()
```

Comment:
Should have the nullable fields here as well with ".optional({ nullable: true })"